tsp-output/@azure-tools/typespec-csharp/src/Generated/Audio.cs

```diff
@@ -309,61 +368,45 @@ internal PipelineMessage CreateCreateSpeechRequest(BinaryContent content, Reques
             request.Headers.Set("Accept", "application/octet-stream");
             request.Headers.Set("Content-Type", "application/json");
             request.Content = content;
+            message.Apply(options);
```

Comment:
The call to `message.Apply(options)` needs to move to the end of the request-creation helper method so that the semantics of `SetHeader` on RequestOptions are correct.  This is because anything the user called `SetHeader` with should replace what the generated code does.

Reply:
https://github.com/Azure/autorest.csharp/issues/4484